contracts/chain-adapters/Arbitrum_Adapter.sol

```diff
@@ -17,6 +17,17 @@ interface ArbitrumL1InboxLike {
         uint256 gasPriceBid,
         bytes calldata data
     ) external payable returns (uint256);
+
+    function createRetryableTicketNoRefundAliasRewrite(
```

Comment:
nit: should we use unsafeCreateRetryableTicket instead (I think the signature is the same)?

According to the [current Inbox implementation on mainnet](https://etherscan.io/address/0x5aed5f8a1e3607476f1f81c3d8fe126deb0afe94#code), createRetryableTicketNoRefundAliasRewrite is deprecated in favor of unsafeCreateRetryableTicket.

Reply:
This is a great call. `createRetryableTicketNoRefundAliasRewrite` calls `unsafeCreateRetryableTicket` so I agree we don't need to change deployed version